lib/device/DeviceDescription.hpp

```diff
@@ -22,5 +28,11 @@ class DeviceDescription {
                              const std::vector<StripDescription> strips);
 
   uint8_t GetLedCount() const;
+
+  static constexpr uint32_t kCheckValue = 0x12345678;
+
+  uint32_t check_value = kCheckValue;
+
+  static constexpr uint32_t kMaxSize = 128;
```

Comment:
This is the number of bytes reserved for the device description to fit in?

Reply:
Correct. There's a compile-time assertion in `Devices.hpp` that the current device fits into this limit.